Error if struct member name contains underscore

The error below was thrown during compilation of the generated source code. Removing the underscores in member names fixed it.

```
[INFO] Compiling 22 source files to D:\Diverse\Idea\netcalc\target\netcalc-generated\java\btc.cab.netcalc\target\classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /D:/Diverse/Idea/netcalc/target/netcalc-generated/java/btc.cab.netcalc/src/main/java/com/btc/cab/netcalc/protobuf/TypesCodec.java:[236,17] cannot find symbol
  symbol:   method setP_mw(java.lang.Double)
  location: variable builder of type com.btc.cab.netcalc.protobuf.Types.ComplexPower.Builder
[ERROR] /D:/Diverse/Idea/netcalc/target/netcalc-generated/java/btc.cab.netcalc/src/main/java/com/btc/cab/netcalc/protobuf/TypesCodec.java:[237,17] cannot find symbol
  symbol:   method setQ_mvar(java.lang.Double)
  location: variable builder of type com.btc.cab.netcalc.protobuf.Types.ComplexPower.Builder
[ERROR] /D:/Diverse/Idea/netcalc/target/netcalc-generated/java/btc.cab.netcalc/src/main/java/com/btc/cab/netcalc/protobuf/TypesCodec.java:[246,17] cannot find symbol
  symbol:   method setMagnitude_kv(java.lang.Double)
  location: variable builder of type com.btc.cab.netcalc.protobuf.Types.ComplexVoltage.Builder
[ERROR] /D:/Diverse/Idea/netcalc/target/netcalc-generated/java/btc.cab.netcalc/src/main/java/com/btc/cab/netcalc/protobuf/TypesCodec.java:[247,17] cannot find symbol
  symbol:   method setAngle_degree(java.lang.Double)
  location: variable builder of type com.btc.cab.netcalc.protobuf.Types.ComplexVoltage.Builder
[ERROR] /D:/Diverse/Idea/netcalc/target/netcalc-generated/java/btc.cab.netcalc/src/main/java/com/btc/cab/netcalc/protobuf/TypesCodec.java:[775,33] cannot find symbol
  symbol:   method getP_mw()
  location: variable typedData of type com.btc.cab.netcalc.protobuf.Types.ComplexPower
[ERROR] /D:/Diverse/Idea/netcalc/target/netcalc-generated/java/btc.cab.netcalc/src/main/java/com/btc/cab/netcalc/protobuf/TypesCodec.java:[776,35] cannot find symbol
  symbol:   method getQ_mvar()
  location: variable typedData of type com.btc.cab.netcalc.protobuf.Types.ComplexPower
[ERROR] /D:/Diverse/Idea/netcalc/target/netcalc-generated/java/btc.cab.netcalc/src/main/java/com/btc/cab/netcalc/protobuf/TypesCodec.java:[786,41] cannot find symbol
  symbol:   method getMagnitude_kv()
  location: variable typedData of type com.btc.cab.netcalc.protobuf.Types.ComplexVoltage
[ERROR] /D:/Diverse/Idea/netcalc/target/netcalc-generated/java/btc.cab.netcalc/src/main/java/com/btc/cab/netcalc/protobuf/TypesCodec.java:[787,41] cannot find symbol
  symbol:   method getAngle_degree()
  location: variable typedData of type com.btc.cab.netcalc.protobuf.Types.ComplexVoltage
```

--- a/com.btc.serviceidl/src/com/btc/serviceidl/generator/cpp/ProxyGenerator.xtend
+++ b/com.btc.serviceidl/src/com/btc/serviceidl/generator/cpp/ProxyGenerator.xtend
@@ -151,13 +151,13 @@ class ProxyGenerator extends BasicCppGenerator {
                     // handle [out] parameters
                     «FOR param : output_parameters»
                        «IF com.btc.serviceidl.util.Util.isSequenceType(param.paramType)»
-                          «typeResolver.resolveDecode(paramBundle, param.paramType, interface_declaration)»( concreteResponse.«param.paramName.toLowerCase»(), «param.paramName» );
+                          «typeResolver.resolveDecode(paramBundle, param.paramType, interface_declaration)»( concreteResponse.«param.paramName.asCppProtobufName»(), «param.paramName» );
                        «ELSE»
-                          «param.paramName» = «makeDecodeResponse(param.paramType, interface_declaration, param.paramName.toLowerCase)»
+                          «param.paramName» = «makeDecodeResponse(param.paramType, interface_declaration, param.paramName.asCppProtobufName)»
                        «ENDIF»
                     «ENDFOR»
                  «ENDIF»
-                 return «makeDecodeResponse(function.returnedType, interface_declaration, function.name.toLowerCase)»
+                 return «makeDecodeResponse(function.returnedType, interface_declaration, function.name.asCppProtobufName)»
                  // decode response <--
               } )«IF function.isSync».Get()«ENDIF»;
            «ENDIF»

--- a/com.btc.serviceidl/src/com/btc/serviceidl/generator/cpp/ProxyGenerator.xtend
+++ b/com.btc.serviceidl/src/com/btc/serviceidl/generator/cpp/ProxyGenerator.xtend
@@ -127,14 +127,14 @@ class ProxyGenerator extends BasicCppGenerator {
                     «val is_failable = com.btc.serviceidl.util.Util.isFailable(param.paramType)»
                     «val protobuf_type = typeResolver.resolveProtobuf(ulimate_type, ProtobufType.RESPONSE).fullyQualifiedName»
                     «typeResolver.resolveCodecNS(paramBundle, ulimate_type, is_failable, Optional.of(interface_declaration))»::Encode«IF is_failable»Failable«ENDIF»< «resolve(ulimate_type)», «IF is_failable»«typeResolver.resolveFailableProtobufType(param.paramType, interface_declaration)»«ELSE»«protobuf_type»«ENDIF» >
-                       ( «resolveSymbol("std::move")»(«param.paramName»), concreteRequest->mutable_«param.paramName.toLowerCase»() );
+                       ( «resolveSymbol("std::move")»(«param.paramName»), concreteRequest->mutable_«param.paramName.asCppProtobufName»() );
                  «ELSEIF com.btc.serviceidl.util.Util.isEnumType(param.paramType)»
-                    concreteRequest->set_«param.paramName.toLowerCase»( «typeResolver.resolveCodecNS(paramBundle, param.paramType)»::Encode(«param.paramName») );
+                    concreteRequest->set_«param.paramName.asCppProtobufName»( «typeResolver.resolveCodecNS(paramBundle, param.paramType)»::Encode(«param.paramName») );
                  «ELSE»
-                    «typeResolver.resolveCodecNS(paramBundle, param.paramType)»::Encode( «param.paramName», concreteRequest->mutable_«param.paramName.toLowerCase»() );
+                    «typeResolver.resolveCodecNS(paramBundle, param.paramType)»::Encode( «param.paramName», concreteRequest->mutable_«param.paramName.asCppProtobufName»() );
                  «ENDIF»
               «ELSE»
-                 concreteRequest->set_«param.paramName.toLowerCase»(«param.paramName»);
+                 concreteRequest->set_«param.paramName.asCppProtobufName»(«param.paramName»);
               «ENDIF»
            «ENDFOR»
            // encode request <--

